engine/src/main/java/org/terasology/physics/bullet/BulletPhysics.java

```diff
@@ -96,59 +96,55 @@
 /**
  * Physics engine implementation using TeraBullet (a customised version of JBullet).
  */
+@RegisterSystem
 public class BulletPhysics implements PhysicsEngine {
+    public static final int AABB_SIZE = Integer.MAX_VALUE;
+
+    public static final float SIMD_EPSILON = 1.1920929E-7F;
```

Comment:
Wut. This constant's name is probably sensible enough, but most people probably won't know what that means anyway (I'm one of them). Might be good with with a one-liner javadoc to explain this field?